resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/ClientArguments.scala

```diff
@@ -66,6 +74,15 @@ private[spark] class ClientArguments(args: Array[String]) {
       throw new IllegalArgumentException("Cannot have primary-py-file and primary-r-file" +
         " at the same time")
     }
+
+    if (verbose) {
+      logInfo("Client arguments for YARN application:")
```

Comment:
I remember we already have such mode in `SparkSubmit`. Can we follow how it's done there? e.g.) https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala#L86-L88

Printing out all args would be enough I suspect.

Reply:
Thanks for your review, your are right, but this is two different --verbose option. SparkSubmit will pass --verbose option to org.apache.spark.deploy.yarn.ClientArgument.scala, which can't handle the --verbose by now. I will add some detailed description of screenshots later.

Reply:
we can also avoid throw Expection by remove --verbose option before pass arguments to org.apache.spark.deploy.yarn.ClientArgument.scala

Reply:
BTW, print all args is enough indeed